[docs] Explain downgrades aren't supported. (#13678)
yugabyte/yugabyte-db: docs/content/preview/manage/upgrade-deployment.md

{{< note title="Note" >}}
Concurrent operations in a cluster can lead to various transactional conflicts, catalog version mismatches, and read restart errors. This is expected, and should be addressed by re-running `upgrade_ysql`.
{{< /note >}}

{{< note title="Note" >}}

> Downgrades are currently unsupported. Please see issue [#13686](https://github.com/yugabyte/yugabyte-db/issues/13686) on Github.

Reviewer: It's better not to have several note elements one right after the other. I suggest just unwrapping both notes (leave the text).
Also, suggestion:
Downgrades are not currently supported.

> {{< note title="Note" >}}
> Concurrent operations in a cluster can lead to various transactional conflicts, catalog version mismatches, and read restart errors. This is expected, and should be addressed by re-running `upgrade_ysql`.
> {{< /note >}}
> 
> {{< note title="Note" >}}
> Downgrades are currently unsupported. Please see issue [#13686](https://github.com/yugabyte/yugabyte-db/issues/13686) on Github.
> {{< /note >}}

Reviewer 2: ```suggestion
{{< note title="Notes" >}}
Concurrent operations in a cluster can lead to various transactional conflicts, catalog version mismatches, and read restart errors. This is expected, and should be addressed by re-running `upgrade_ysql`.

Downgrades aren't currently supported. This is tracked in GitHub issue [#13686](https://github.com/yugabyte/yugabyte-db/issues/13686).
{{< /note >}}
```
